response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -60,15 +60,11 @@ def get_collection_exercise(short_name, period):
     return response.json()
 
 
-def get_collection_exercise_event_page_info(short_name, period):
-    logger.debug(
-        "Retrieving collection exercise details for the event page",
-        short_name=short_name,
-        period=period,
-    )
-    url = (
-        f'{app.config["BACKSTAGE_API_URL"]}/v1/collection-exercise/{short_name}/{period}/events'
-    )
+def get_collection_exercise_events(short_name, period):
+    logger.debug('Retrieving collection exercise details for the event page',
+                 short_name=short_name, period=period)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/collection-exercise/{short_name}/{period}/events'
+
     response = requests.get(url)
     if response.status_code != 200:
```

Comment:
I know its not your code, but this is probably better standard if we do:

Reply:
@shahi645 this is being updated in #195 & co.